.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: go
+env:
+  global:
+    - INPUT_PLATFORM=linux/amd64  # ,linux/arm64
```

Comment:
The `linux/arm64` target is commented out. Is it correct?

Reply:
Yes, we don't have a SaaS running on arm64 yet, so I think it would be a waste of resources